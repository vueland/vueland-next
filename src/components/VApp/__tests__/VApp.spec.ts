import { mount } from '@vue/test-utils'
import { VApp } from '../'

describe('VApp', () => {
  test('should mount component and match snapshot', () => {
    const wrap = mount(VApp, {
      slots: {
        default: '<div class="default"></div>'
      }
    })

    expect(wrap.find('.default').exists()).toBe(true)
    expect(wrap.html()).toMatchSnapshot()
  })
})
