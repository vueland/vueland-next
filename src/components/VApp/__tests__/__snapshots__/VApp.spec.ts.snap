// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VApp should mount component and match snapshot 1`] = `
<div class="v-app">
  <div class="default">
  </div>
</div>
`;
